src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -488,6 +495,28 @@ private void Log<T>(string xpath, T res)
                 _logger.Info($"Setting: {xpath} = {res}");
             }
         }
+
+        public TestOutcome MapWarningOutcome(string outcome)
+        {
+            if (outcome == null)
+                return TestOutcome.Skipped;
+            var testOutcome = outcome.ToLower() switch
+            {
+                "skipped" => TestOutcome.Skipped,
+                "failed" => TestOutcome.Failed,
+                "passed" => TestOutcome.Passed,
+                "none" => TestOutcome.None,
+                _ => TestOutcome.NotFound,
+            };
+            if (testOutcome == TestOutcome.NotFound)
+            {
+                _logger.Warning(
+                    $"Invalid value ({outcome}) for MapWarningTo, should be either Skipped,Failed,Passed or None");
+                return TestOutcome.Skipped;
+            }
```

Comment:
Same recommendation, Enum.TryParse with ignoreCase: true.

Reply:
And thanks again!